runtime/v2/shim_unix.go

```diff
@@ -30,7 +30,7 @@ import (
 )
 
 func openShimLog(ctx context.Context, bundle *Bundle, _ func(string, time.Duration) (net.Conn, error)) (io.ReadCloser, error) {
-	return fifo.OpenFifo(ctx, filepath.Join(bundle.Path, "log"), unix.O_RDONLY|unix.O_CREAT|unix.O_NONBLOCK, 0700)
+	return fifo.OpenFifo(ctx, filepath.Join(bundle.Path, "log"), unix.O_RDWR|unix.O_CREAT|unix.O_NONBLOCK, 0700)
```

Comment:
I don't remember why not use RDWR before. ping @AkihiroSuda and @crosbymichael

Reply:
found it https://github.com/containerd/containerd/pull/3266

I think add `f.Close()` is fine.